RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -350,33 +350,55 @@ class RNUnifiedContacts: NSObject {
     return emailAddresses
   }
   
-  func transformNumber(number: NSDictionary) -> CNLabeledValue {
+  func convertPhoneNumberToNSDictionary(phoneNumber: NSDictionary) -> CNLabeledValue {
     var label = String()
-    if (number["label"] as! String == "home") {
-      label = CNLabelHome
-    } else if (number["label"] as! String == "work") {
-      label = CNLabelWork
-    } else if (number["label"] as! String == "mobile") {
-      label = CNLabelPhoneNumberMobile
+    switch (phoneNumber["label"] as! String) {
+      case "home":
+        label = CNLabelHome
+      case "work":
+        label = CNLabelWork
+      case "mobile":
+        label = CNLabelPhoneNumberMobile
+      case "iPhone":
+        label = CNLabelPhoneNumberiPhone
+      case "main":
+        label = CNLabelPhoneNumberMain
+      case "home fax":
+        label = CNLabelPhoneNumberHomeFax
+      case "work fax":
+        label = CNLabelPhoneNumberWorkFax
+      case "pager":
+        label = CNLabelPhoneNumberPager
+      case "other":
+        label = CNLabelOther
+      default:
+        label = ""
     }
     
     return CNLabeledValue(
       label:label,
-      value:CNPhoneNumber(stringValue: number["number"] as! String)
+      value:CNPhoneNumber(stringValue: phoneNumber["number"] as! String)
     )
   }
   
-  func transformEmail(email: NSDictionary) -> CNLabeledValue {
+  func convertEmailAddressToNSDictionary(emailAddress: NSDictionary) -> CNLabeledValue {
     var label = String()
-    if (email["label"] as! String == "home") {
-      label = CNLabelHome
-    } else if (email["label"] as! String == "work") {
-      label = CNLabelWork
+    switch (emailAddress["label"] as! String) {
+      case "home":
+        label = CNLabelHome
+      case "work":
+        label = CNLabelWork
+      case "iCloud":
+        label = CNLabelEmailiCloud
+      case "other":
+        label = CNLabelOther
+      default:
```

Comment:
Same here as with Phone Numbers. Should we allow custom labels for email addresses?